lib/webenginesettings.h

```diff
@@ -25,6 +26,12 @@ class WebEngineSettings : public QMozEngineSettings {
 
     explicit WebEngineSettings(QObject *parent = 0);
     virtual ~WebEngineSettings();
+
+private:
+    QScopedPointer<WebEngineSettingsPrivate> d_ptr;
+
+    inline void setSearchEngine();
+    inline void handleObserve(const QString &message, const QVariant &data);
```

Comment:
Both of these could go to private class I think. Why inlining?

Reply:
Just an overly defensive habit, they're private members so the symbol visibility should also be private. Inline symbols aren't exported outside of the compilation unit. It's equivalent to a static non-member function or anonymous namespace in the .cpp file.

It's a little less over the top here though because WebEngineSettings should be an explicitly exported class so it also prevents the symbols being exported from the library. It seems no-ones configured explicit exports for this library though so everything is exported unless told otherwise.

Reply:
Putting them in the private class means the private class then needs a pointer back to the public class which is just more more complexity for no return.

Reply:
Thanks for clarification.